sync/sync-impl/src/main/res/layout/activity_enter_code.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2023 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+              xmlns:app="http://schemas.android.com/apk/res-auto"
+              android:orientation="vertical"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent">
+
+    <include
+        android:id="@+id/includeToolbar"
+        layout="@layout/include_default_toolbar"/>
+
+    <FrameLayout
+        android:id="@+id/enterCodeContainer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@+id/includeToolbar"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent">
+        <com.duckduckgo.mobile.android.ui.view.text.DaxTextView
+            android:id="@+id/enterCodeHint"
+            android:layout_width="310dp"
+            android:layout_height="214dp"
+            android:layout_margin="@dimen/keyline_6"
+            android:gravity="center"
+            app:typography="caption"
+            app:textType="secondary"
+            android:text="@string/sync_enter_code_hint"/>
+        <com.duckduckgo.mobile.android.ui.view.text.DaxTextView
+            android:id="@+id/pastedCode"
+            android:layout_width="310dp"
```

Comment:
add this width and height to the `FrameLayout` to avoid duplication. Both `DaxTextView` can then `match_parent`